htdocs/fichinter/card-rec.php

```diff
@@ -102,13 +104,13 @@
 
 // Create predefined intervention
 if ($action == 'add') {
-	if (!GETPOST('titre')) {
+	if (!GETPOSTISSET('titre')) {
```

Comment:
GETPOSTISSET seems not enough GETPOST must also return a value.
What's wrong with GETPOST('titre') ?
If post is not defined, GETPOST should return an empty value without warning too.